spec/lib/portus/deprecation_error_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+describe Portus::DeprecationError do
+  let(:message) { "example" }
+  subject(:error) { described_class.new(message) }
+
+  context "#to_s" do
+    subject { error.to_s }
```

Comment:
I think it would be simpler to have just one test like this:

```ruby
it "prefixes the message with [DEPRECATED]" do
  is_expected.to eq("[DEPRECATED] #{message}")
end
```

Reply:
every project is different :)